app/controllers/account/dashboard_controller.rb

```diff
@@ -3,5 +3,18 @@ class Account::DashboardController < ApplicationController
 
   def index
     @chart_value = current_companies
+    @count_of_pages = todays_pages
+  end
+
+  private
+
+  def todays_pages
+    count = 0
+    current_companies.each do |company|
```

Comment:
what if you have 100000 companies in DB?

Use one line of code to count all pages for all companies!!!